generator/src/main/java/com/scottlogic/deg/generator/walker/ReductiveDecisionTreeWalker.java

```diff
@@ -50,9 +54,17 @@ public ReductiveDecisionTreeWalker(
             return reductiveRowSpecGenerator.createRowSpecsFromFixedValues(reductiveState, constraintNode);
         }
 
+        Iterator<Object> valueIterator = reductiveState.getValuesFromLastFixedField().iterator();
```

Comment:
For debugging, check to see if there are any values in the last fixed field.
If there are no values then inform the monitor, otherwise proceed. Requires the stream to be 'converted' to an iterator (to call `hasNext`) and then back again, so `flatMap` can be used.